docs/wasmer-js/client/examples/transforming-wasi-modules/transforming-wasi-modules.md

```diff
@@ -1,47 +1,35 @@
 ---
-id: wasmer-js-modules-transforming-wasi-modules
+id: wasmer-js-client-transforming-wasi-modules
 title: Wasmer-JS in the Browser
 sidebar_label: Transforming WASI Modules
 ---
 
-[Full Example Project Source Code](https://github.com/wasmerio/docs.wasmer.io/tree/master/docs/wasmer-js/node-modules/examples/transforming-wasi-modules)
+[Full Example Project Source Code](https://github.com/wasmerio/docs.wasmer.io/tree/master/docs/wasmer-js/client/examples/transforming-wasi-modules)
 
-# Why Is Transformation Necessary?
-
-In the previous Hello World example, we showed you how to run the very basic `as-echo` WASM module that received a text string as an argument and simply echoed it back via standard out.  However, some WASI modules may be compiled in a way that means they can't immediately be run from a JavaScript environment such as a browser.
-
-For example, any module that calls the [clock\_time\_get](https://github.com/WebAssembly/WASI/blob/master/phases/snapshot/docs.md#-clock_time_getid-clockid-precision-timestamp---errno-timestamp) WASI API, must be able to supply a 64-bit, signed integer.  However, passing a JavaScript `BigInt` to a WebAssembly `i64` is not yet supported &mdash; this detail is still at the proposal stage.  (See [here](https://github.com/WebAssembly/JS-BigInt-integration/issues/15) and [here](https://github.com/WebAssembly/proposals/issues/7) for details).
-
-However, it is not impossible to run such a module; but before we can, we must first ***transform*** it using `@wasmer/wasm-transformer`.
-
-> ### Under The Hood  
-> Technically, this the purpose of this transformation process is to adapt the WebAssembly interface so that it can send and receive JavaScript `BigInt`s (64-bit, signed integers).
->
-> No data loss occurs here because a JavaScript `BigInt` is transformed into a `Uint8Array` containing 8, unsigned, 8-bit integers.
+# Transforming WASI Modules in the Browser
 
+Irrespective of whether your JavaScript code runs on the client or the server, the statement shown below will be needed to transform a WASI module.
```

Comment:
Can we link to the document that you wrote here? I feel like this is one of the cases where this would come up like you mentioned in #23 😄

Reply:
We can do this in a follow up PR, as I want to merge your serverside improvements ASAP 😄 👍